src/features/auth/components/Screen/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Layout, Footer } from '../../../../common/components';
+
+import Header from '../Header';
+
+const Screen = ({ children }) => (
```

Comment:
at first i was like "what is this"? so maybe a Screen is not the best name. Or is it? What about ScreenLayout or ScreenWrapper ?

Reply:
1. isnt it redundant to prefix with Auth since its in the auth folder? 
2. public would work only if we rename auth to public - thats actually something we want to do right? we could do it in this PR
3. meh

is `ScreenLayout` or `ScreenWrapper` not good? since its scoped in the `auth` folder its clear what it is wrapping

Reply:
ad 1. What about AuthHeader then? It's common when we are defining how the X (generic header) should look in Y (feature auth). so we get specific AuthHeader.

Public prefix - I would drop it completely... why keeping it?

Reply:
best would be Page as page has header/body/footer.
Children in this case are the body.